addons/website_sale/models/product.py

```diff
@@ -451,13 +504,39 @@ class Product(models.Model):
 
     website_url = fields.Char('Website URL', compute='_compute_product_website_url', help='The full URL to access the document through the website.')
 
+    base_unit_count = fields.Float('Base Unit Count', required=True, default=1, help="Display base unit price on your eCommerce pages. Set to 0 to hide it for this product.")
```

Comment:
One day we should really split product/product_template in two files, this is very misleading, you scroll of ctrl+f a method but you never know in which model you are, not related to this pr tho..